mindsdb/integrations/handlers/auto_ts_handler/README.md

```diff
@@ -0,0 +1,100 @@
+### Briefly describe what ML framework does this handler integrate to MindsDB, and how?
+Auto_timeseries is a complex model building utility for time series data. Since it automates many 
+Tasks involved in a complex endeavor, it assumes many intelligent defaults. But you can change them. 
+Auto_Timeseries will rapidly build predictive models based on Statsmodels ARIMA, Seasonal ARIMA, Prophet 
+and Scikit-Learn ML. It will automatically select the best model which gives best score specified.
+
+
+
+Call this handler with
+`USING ENGINE='auto_ts'` 
+
+### Why is this integration useful? What does the ideal predictive use case for this integration look like? When would you definitely not use this integration?
+This integration is useful for time series data. It will automatically select the best model which gives best score specified. And it integrations with best Time Series 
+libraries like Statsmodels ARIMA, Seasonal ARIMA, Prophet and Scikit-Learn ML.
+
+The ideal use case is a very large dataset with many predictive features, where the user doesn't have strong priors about how these features may affect the target variable.
```

Comment:
Is this true? Is the user to assume that exogenous variable support is first class, then?